lisa/runner.py

```diff
@@ -81,19 +81,10 @@ async def start(self) -> None:
         await super().start()
 
         self._initialize_runners()
-
-        try:
-            raw_results: List[List[TestResult]] = run_in_threads(
-                [
-                    partial(runner.run, id_=runner.type_name())
-                    for runner in self._runners
-                ],
-                completed_callback=self._completed_callback,
-                log=self._log,
-            )
-        finally:
-            for runner in self._runners:
-                runner.close()
+        raw_results: List[List[TestResult]] = []
```

Comment:
Another way is to move the `if` sentence into _start_run. And if runner is empty, it returns an empty list.